monosize.config.js

```diff
@@ -1,6 +1,12 @@
 const path = require('path');
 
 module.exports = {
+  repository: 'https://github.com/microsoft/griffel',
+  storage: require('monosize-storage-upstash')({
+    url: 'https://us1-nearby-hyena-35747.upstash.io',
+    readonlyToken:
+      'AoujASQgNGQwNGI0YjAtNmExNy00MzNjLWIwNTQtMTIyMGRlODQwZDk2QaCJNVtEfHj6pDLLscOcgMEt-Be1VWcMYbMRR_EqVCw=',
+  }),
```

Comment:
This is an example of adapter for storage. With this config it's possible to compare results with `upstream/main` 🚀